### ISSUE TITLE
Instead of default value make default value as device name

### ISSUE THREAD

**Issue by Chromicle** [opendatakit/skunkworks-crow#201]
#### Software and hardware versions 
Android Samsung J8

#### Problem description
Currently, the default value of hotspot SSID is "hotspot" only instead of that make the default value as a device name i.e the device name which app is running
for every device, it is a unique name so using that name instead of the default value it gives the good user experience

#### Steps to reproduce the problem
Not an issue new feature

#### Expected behavior
The default name(hotspot) should be removed instead of that should come device name

**Comment by Chromicle** [comment]
@shobhitagarwal1612 can I work on this, Thankyou!

**Comment by lakshyagupta21** [comment]
Currently, we don't even use that, the option to modify is there but we haven't added the functionality to use the user-defined hotspot ID, its still in backlog and the issue is already been raised in #153. Maybe after its done we can go ahead with this issue.

**Comment by Chromicle** [comment]
ok! after completing that issue I will do this

**Comment by Chromicle** [comment]
I will claim that issue an I will complete that

**Comment by shobhitagarwal1612** [comment]
I think a good way to differentiate multiple devices' hotspot running this app would be to something like `crow-<device_name>` eg, `crow-xiaomi`

**Comment by Chromicle** [comment]
> I think a good way to differentiate multiple devices' hotspot running this app would be to something like `crow-<device_name>` or `skunkworks-crow-xiaomi`

Yeah, it is better to be like  `crow-<device_name>` in case my phone what would be better of these names  `crow-SM-J810G` or `crow-Samsung SM-J810G` or `crow-Samsung`  
SM-J810G is my phone model name

**Comment by devanshi7799** [comment]
Is this issue solved? Or else i would like to work on this

**Comment by lakshyagupta21** [comment]
@devanshi7799 As @Chromicle also shown interest in working on this issue, if he is fine with assigning this to you, then you can work on this.

**Comment by Chromicle** [comment]
As discussed before we need discussion for this so, I did not submitted PR I will work on this
